src/app/map-viewer/map-viewer.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Map } from 'leaflet';
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs/Subject';
+import { Observable } from 'rxjs/Rx';
+
+@Injectable()
+export class MapViewerService {
+  public map: Map;
+  public baseMaps: any;
+  public currentLat: number;
+  public currentLng:  number;
+  public altitude: number;
+  public accuracy: number;
+  hasLocation = false;
+  public center: any;
+
+  locationSource = new Subject<any>();
+
+  geoOptions = {
+      enableHighAccuracy: true,
+      maximumAge: 30000,
+      timeout: 27000
+  };
+  constructor() {
+    this.hasLocation = false;
+      if (navigator.geolocation) {
```

Comment:
No. We don't need an avatar section in About Us page;  their avatars will be displayed inside the map.
Volunteers page is fine. Only developers for c4sg web app will be in this map. 
Perhaps, We could show a different map for every project.